test/Microsoft.ML.CpuMath.UnitTests.DotNetCoreApp/UnitTests.cs

```diff
@@ -0,0 +1,451 @@
+using System;
```

Comment:
All code files need a copyright header on them.  See other `.cs` files for what the header should look like.

Reply:
Thank you.  Some new files include the header, while some others don't.  Will fix it right away.